guide/interactions/context-menus.md

```diff
@@ -0,0 +1,43 @@
+# Context Menus
+
+Context Menus are application commands which appear on right clicking or tapping a user or a message.
```

Comment:
```suggestion
Context Menus are application commands which appear on right clicking or tapping an user or a message.
```

Reply:
I think it's right as it is, just "a user". If I'm not wrong, "an" is used when the word starts with a vowel sound. It's a little bit confusing because "user" starts with "u", but when you pronounce "user" it starts with a "y".

Reply:
then just ignore it